src/test/resources/Features/UI/surveillanceComplaints.feature

```diff
@@ -68,7 +68,7 @@ Scenario: Associate a criteria to a complaint
     Given I'm logged in as "ROLE_ACB"
     And I am on complaints page
     And I click Add New Complaint button
-    And I enter complaint data in fields: "SLI Compliance" "07/12/2019" "07/12/2019" "ONCACB005" "ONC001" "Developer" "TBD"
+    And I enter complaint data in fields: "SLI Compliance" "07/12/2019" "" "ONCACB005" "ONC001" "Developer" "TBD"
```

Comment:
I think it would make more sense to remove the "close date" field entirely, as opposed to setting it to blank. The only reason to not do so is if the method that handles this step also handles edit, and so needs that field, but I don't know if that's the case

Reply:
I realized that right after creating PR and updated tests to remove closed date from add complaint step